howso/utilities/utilities.py

```diff
@@ -575,10 +571,10 @@ def validate_features(features: Mapping[str, Union[FeatureAttributes, Mapping]],
         valid_feature_types += list(extended_feature_types)
 
     for f_name, f_desc in features.items():
-        if isinstance(f_desc, FeatureAttributes):
-            f_type = f_desc.type
-        else:
+        if isinstance(f_desc, Dict):
             f_type = f_desc.get("type")
+        else:
+            f_type = f_desc.type
         if f_type not in valid_feature_types:
             raise ValueError(f"The feature name '{f_name}' has invalid "
                              f"feature type - '{f_type}'")
```

Comment:
```suggestion
    for f_name, f_desc in features.items():
        f_type = f_desc.get("type")
        if f_type not in valid_feature_types:
            raise ValueError(f"The feature name '{f_name}' has invalid "
                             f"feature type - '{f_type}'")
```

Reply:
Should be addressed by: d66654ea10038f70a275bb4aa504128552745e60